Formula/poac.rb

```diff
@@ -0,0 +1,37 @@
+class Poac < Formula
+  desc "Package manager for C++"
+  homepage "https://github.com/poacpm/poac"
+  url "https://github.com/poacpm/poac.git",
+    tag:      "0.3.4",
+    revision: "f905a175ccd1b9ad94041e1e0a29d12d92e8308d"
```

Comment:
```suggestion
  url "https://github.com/poacpm/poac.git",
      tag:      "0.3.4",
      revision: "f905a175ccd1b9ad94041e1e0a29d12d92e8308d"
```
Do we need a git clone? Can we use a tarball instead?

Reply:
@carlocab 

Thank you for your review!

> Do we need a git clone? Can we use a tarball instead?

Yes, it would be through GitHub, but do you prefer it rather than a git clone?

https://github.com/poacpm/poac/archive/refs/tags/0.3.4.tar.gz

Reply:
Yes please. Tarballs are faster/smaller and better for caching.

Reply:
I see. Sure :)

Reply:
@carlocab 

Should I keep the following line or remove it?

```rb
head "https://github.com/poacpm/poac.git", branch: "main"
```

Reply:
You can keep it, as long as you know `brew install --HEAD poac` works (even if only for now).

Reply:
Thank you!